tests/lib/prepare.sh

```diff
@@ -362,11 +362,22 @@ repack_snapd_snap_with_deb_content_and_firstboot_tweaks() {
     cp /usr/lib/snapd/info "$UNPACK_DIR"/usr/lib/
 
     # now install a unit that setups enough so that we can connect
-    
-    # TODO:UC20: use something other than "snapd.core-fixup.sh"
-    # (ok for now because nothing will happen with snapd.core-fixup.sh on UC20)
+    cat > "$UNPACK_DIR"/lib/systemd/system/snapd.spread-tweaks.service <<'EOF'
```

Comment:
spread-tests-run-mode-tweaks.service

Reply:
Thanks, updated to "snapd.spread-tests-run-mode-tweaks.service" (the snap. prefix is important because that's what snapd will automatically enable during seeding).

Reply:
yes, sorry I didn't mean to drop the prefix